packages/c/crashpad/xmake.lua

```diff
@@ -0,0 +1,27 @@
+package("crashpad")
+    set_homepage("https://chromium.googlesource.com/crashpad/crashpad/+/refs/heads/main/README.md")
+    set_description("Crashpad is a crash-reporting system.")
+
+    if is_host("windows") then
+        if is_arch("x64", "x86_64") then
+            set_urls("http://get.backtrace.io/crashpad/builds/crashpad-release-x86-64-$(version).zip")
+            add_versions("stable", "b3facf8a802dfd12daf4d9fba416f4d4b5df0ae544afa14080662fa978aa18cb")
+            add_versions("latest", "7705073dfff89c376303cacea3a6f8c63322f77566ad5cdbe37060cf3cef9f8b")
```

Comment:
incorrect version, please use a fake version with commit date, like "2021.1.1" instead of `stable` and `latest`

Reply:
```lua
set_urls("http://get.backtrace.io/crashpad/builds/crashpad-release-x86-64-$(version).zip", function (version)
    local map = {
       ["2021.1.1"] = "stable",
       ...
    }
    return map[tostring(version)]
end)
```

Reply:
I implemented the suggestion but am getting a weird error now